.php_cs

```diff
@@ -17,18 +17,181 @@ $finder = PhpCsFixer\Finder::create()
+        'no_trailing_comma_in_list_call'              => true,
+        'no_trailing_comma_in_singleline_array'       => true,
+        'no_trailing_whitespace'                      => true,
+        'no_trailing_whitespace_in_comment'           => true,
+        'no_unneeded_control_parentheses'             => ['statements' => ['break', 'clone', 'continue', 'echo_print', 'return', 'switch_case', 'yield']],
+        'no_unneeded_curly_braces'                    => true,
+        'no_unneeded_final_method'                    => true,
+        'no_unreachable_default_argument_value'       => true,
+        'no_unused_imports'                           => true,
+        'no_useless_else'                             => true,
+        'no_useless_return'                           => false,
+        'no_whitespace_before_comma_in_array'         => true,
+        'no_whitespace_in_blank_line'                 => true,
+        'non_printable_character'                     => ['use_escape_sequences_in_strings' => true],
+        'normalize_index_brace'                       => true,
+        'not_operator_with_space'                     => false,
+        'not_operator_with_successor_space'           => false,
+        'object_operator_without_whitespace'          => true,
+        'ordered_class_elements'                      => false,
+        'ordered_imports'                             => true,
+        'php_unit_construct'                          => ['assertions' => ['assertEquals', 'assertSame', 'assertNotEquals', 'assertNotSame']],
+        'php_unit_dedicate_assert'                    => ['target' => 'newest'],
+        'php_unit_expectation'                        => ['target' => 'newest'],
+        'php_unit_fqcn_annotation'                    => true,
+        'php_unit_mock'                               => ['target' => 'newest'],
+        'php_unit_namespaced'                         => ['target' => 'newest'],
+        'php_unit_no_expectation_annotation'          => ['target' => 'newest', 'use_class_const' => true],
+        'php_unit_strict'                             => ['assertions' => ['assertAttributeEquals', 'assertAttributeNotEquals', 'assertEquals', 'assertNotEquals']],
+        'php_unit_test_annotation'                    => ['case' => 'camel', 'style' => 'prefix'],
+        'php_unit_test_class_requires_covers'         => false,
+        'phpdoc_add_missing_param_annotation'         => ['only_untyped' => false],
+        'phpdoc_align'                                => ['tags' => ['param', 'return', 'throws', 'type', 'var']],
+        'phpdoc_annotation_without_dot'               => false,
+        'phpdoc_indent'                               => true,
+        'phpdoc_inline_tag'                           => true,
+        'phpdoc_no_access'                            => true,
+        'phpdoc_no_alias_tag'                         => ['property-read' => 'property', 'property-write' => 'property', 'type' => 'var'],
```

Comment:
This is almost like the default options included in Symfony, but is missing the `'link' => 'see'` option, that intentional?

Reply:
fixed